VAGRANT.md

```diff
@@ -18,10 +18,47 @@ pouch run -d --name nginx -p 80:80 nginx
 curl http://localhost
 ```
 
-## Build pouch with vagrant
+## Getting Started with Kubernetes
 
 ```bash
+# On MacOS or Linux
+export POUCH_BUILD=true POUCH_KUBE=true
```

Comment:
in order to avoid env pollution, we can compact two lines into one line
```
POUCH_BUILD=true POUCH_KUBE=true vagrant up
```

Reply:
are we need POUCH_BUILD here for `Getting Started with Kubernetes`?

Reply:
apt-get -y install pouch will install (1.0.0.rc1-0~ubuntu), not up to date.